conda/install.py

```diff
@@ -635,8 +642,19 @@ def messages(prefix):
     finally:
         rm_rf(path)
 
+
+def ensure_write(prefix, dist):
+    if not can_open_all_files_in_prefix(prefix, dist['files']):
```

Comment:
Seeing an error here on Windows.  Apparently `dist` isn't what I thought it was.  Investigating.